openshift/patches/0001-Patch-builder-images-in-midstream.patch

```diff
@@ -0,0 +1,90 @@
+ templates/rust/manifest.yaml       |  2 --
+ templates/typescript/manifest.yaml |  2 --
+ 6 files changed, 7 insertions(+), 19 deletions(-)
+ delete mode 100644 templates/go/manifest.yaml
+ delete mode 100644 templates/node/manifest.yaml
+ delete mode 100644 templates/rust/manifest.yaml
+ delete mode 100644 templates/typescript/manifest.yaml
+
+diff --git a/buildpacks/builder.go b/buildpacks/builder.go
+index 072ef677..6e2fa7e6 100644
+--- a/buildpacks/builder.go
++++ b/buildpacks/builder.go
+@@ -21,13 +21,13 @@ import (
+ 
+ // DefaultBuilderImages for Pack builders indexed by Runtime Language
+ var DefaultBuilderImages = map[string]string{
+-	"node":       "gcr.io/paketo-buildpacks/builder:base",
+-	"typescript": "gcr.io/paketo-buildpacks/builder:base",
+-	"go":         "gcr.io/paketo-buildpacks/builder:base",
+-	"python":     "gcr.io/paketo-buildpacks/builder:base",
+-	"quarkus":    "gcr.io/paketo-buildpacks/builder:base",
+-	"rust":       "gcr.io/paketo-buildpacks/builder:base",
+-	"springboot": "gcr.io/paketo-buildpacks/builder:base",
++	"node":       "quay.io/boson/faas-nodejs-builder",
++	"typescript": "quay.io/boson/faas-nodejs-builder",
++	"go":         "quay.io/boson/faas-go-builder",
++	"python":     "quay.io/boson/faas-python-builder",
++	"quarkus":    "quay.io/boson/faas-jvm-builder",
++	"rust":       "quay.io/boson/faas-rust-builder",
++	"springboot": "quay.io/boson/faas-jvm-builder",
+ }
+ 
+ //Builder holds the configuration that will be passed to
+diff --git a/templates/go/manifest.yaml b/templates/go/manifest.yaml
+deleted file mode 100644
+index d8ebe717..00000000
+--- a/templates/go/manifest.yaml
++++ /dev/null
+@@ -1,2 +0,0 @@
+-builders:
+-  default: quay.io/boson/faas-go-builder
+diff --git a/templates/node/manifest.yaml b/templates/node/manifest.yaml
+deleted file mode 100644
+index d8d20ed0..00000000
+--- a/templates/node/manifest.yaml
++++ /dev/null
+@@ -1,2 +0,0 @@
+-builders:
+-  default: quay.io/boson/faas-nodejs-builder
+diff --git a/templates/quarkus/manifest.yaml b/templates/quarkus/manifest.yaml
+index 881fb39e..f93bfaf5 100644
+--- a/templates/quarkus/manifest.yaml
++++ b/templates/quarkus/manifest.yaml
+@@ -1,7 +1,3 @@
+-builders:
+-  default: quay.io/boson/faas-jvm-builder:latest
+-  jvm: quay.io/boson/faas-jvm-builder:latest
+-  native: quay.io/boson/faas-quarkus-native-builder:latest
+ buildEnvs:
+   - name: MAVEN_S2I_ARTIFACT_DIRS
+     value: target
+diff --git a/templates/rust/manifest.yaml b/templates/rust/manifest.yaml
+deleted file mode 100644
+index 1bae3110..00000000
+--- a/templates/rust/manifest.yaml
++++ /dev/null
+@@ -1,2 +0,0 @@
+-buildpacks:
+-  - docker.io/paketocommunity/rust
+diff --git a/templates/typescript/manifest.yaml b/templates/typescript/manifest.yaml
+deleted file mode 100644
+index d8d20ed0..00000000
+--- a/templates/typescript/manifest.yaml
++++ /dev/null
+@@ -1,2 +0,0 @@
+-builders:
+-  default: quay.io/boson/faas-nodejs-builder
+-- 
+2.35.3
```

Comment:
I think this changes could be part of the PR directly not patching it on each sync.